CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing
+
+Implementing and maintaining a protocol stack is a lot of work, therefore any
+help is appreciated, from creating issues, to contributing documentation, fixing
+issues and adding new features.
+
+Please follow the best-practice contribution guidelines as mentioned below when
+submitting any changes.
+
+### Contributions License Agreement
+
+Before contributing, please read the [Contributions License Agreement](./CLA.md).
+Every contributor must sign this agreement before any code is merged into the
+repository.
+
+### Git
+
+Please, do your changes with the develop branch in your forked repo
```

Comment:
This is not very clear, after reading it I am not sure what I am supposed to do...

Is this something you care about or we can just delete this sentence?

Reply:
Was trying to say that it is best practice to do a fork and use an own branch there (and not use main branch in the fork to do changes) ... maybe I should also phrase it that way :-)

Reply:
text adjusted

Reply:
You mean fork or clone? I always clone on my side...
To be honest, I don't know if we should go too much into how each contributor is managing his branches in this file, they are many equivalent ways of doing it.
I think our end goal here is to have clean PR that we can review. As PR reviewers, we can ask to change the branch logic if needed.

We can update this file if there is too much common mistakes.

The main branch is protected by rules: PR with approval required to merge, so that should prevent the worst mistakes.

Reply:
Ok, I remove it.

BTW Would you see any need for a special commit format or such?